kinto/plugins/admin/views.py

```diff
@@ -11,16 +11,10 @@
 # Configured home page
 @cache_forever
 def admin_home_view(request):
-    settings = {
-        'authMethods': aslist(request.registry.settings.get('multiauth.policies'))
-    }
-    globalSettings = '<script>window.globalSettings = {};</script>'.format(json.dumps(settings))
```

Comment:
If we don't need to generate the settings from the installed kinto plugins, we don't need to have a python view anymore to render the html page. (except maybe to route between the index.html page and the help page but do we really?)